src/workers/parser/tests/__snapshots__/closest.spec.js.snap

```diff
@@ -78,7 +78,6 @@ FunctionDeclaration (1:0,3:1)
             right: Identifier (2:13,2:14)
               name: 'n'
     directives: []
-  expression: false
```

Comment:
This field was deleted from Babel 7's AST because it is just duplicate data. If you want to know if the body of an arrow function is an expression, people are expected to check `node.body.type !== "BlockStatement"`